w09h01/test/pgdp/pingutrip/ReadWayPointsTest.java

```diff
@@ -1,6 +1,7 @@
 package pgdp.pingutrip;
 
 import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
How could _that_ happen? 🫣

Reply:
ask @nilsreichardt :)

(no, ping wasn't unnecessary - you can merge)